db/fixtures/chargeback_rates.yml

```diff
@@ -18,7 +18,7 @@
           :fixed_rate: 0.0
           :variable_rate: 0.02
     - :description: Used CPU Cores
-      :group: cpu
+      :group: cpu_container
```

Comment:
Please apply this commit over this PR: https://github.com/zeari/manageiq/commit/347b20cef6a4571759a0f66beb770dec66a9a805

It also changes the group to `cpu_cores` which I think makes more sense here. 
The naming has to be uniform with column fields also changed there.